src/core_ocean/shared/mpas_ocn_vmix.F

```diff
@@ -1093,6 +1095,24 @@ subroutine ocn_vmix_implicit(dt, meshPool, statePool, forcingPool, scratchPool,
          call mpas_timer_stop('CVMix avg')
       endif
 
+      ! if using GOTM, then viscosity has to be averaged from cell centers to cell edges
+      if ( config_use_gotm ) then
+
+         nEdges = nEdgesArray( 1 )
+         call mpas_timer_start('GOTM avg', .false.)
+         !$omp do schedule(runtime) private(cell1, cell2, k)
+         do iEdge=1,nEdges
+            vertViscTopOfEdge(:, iEdge) = 0.0_RKIND
+            cell1 = cellsOnEdge(1,iEdge)
+            cell2 = cellsOnEdge(2,iEdge)
+            do k=1,maxLevelEdgeTop(iEdge)
```

Comment:
```suggestion
            do k=minLevelEdgeBot(iEdge),maxLevelEdgeTop(iEdge)
```

Reply:
@cbegeman Thanks for these suggestions! Just want to make sure should we use `minLevelEdgeBot` or `minLevelEdgeTop` here? I notice that a few lines above this when CVMix is used, the `k` loop starts from `minLevelEdgeTop`. Although it probably does not matter as I think we will not use the value of `vertViscTopOfEdge` at the top of the top active layer, right? So I guess we should use `minLevelEdgeBot` for CVMix as well?

Reply:
@qingli411 That's a great question. I think you're right that we should use `minLevelEdgeBot` for CVMix above. I'll test/include that change in my next `minLevel*` PR so you only need to make the change in the GOTM section.

Reply:
@qingli411, `minLevelEdgeBot` is the deeper of the two `minLevelCell` on the edge, and `minLevelEdgeTop` is the shallower of the 2.  So they are both 1 for `minLevelCell=1` everywhere.

Reply:
Thanks, @xylar! OK. I misunderstood it... Now it makes sense to me.

Reply:
@qingli411 Glad you asked. `minLevelEdgeBot` is actually 1. It's defined as `max(minLevelCell(cell1),minLevelCell(cell2))` so in the case where there are no inactive cells `minLevelCell(cell1)=minLevelCell(cell2)=1`. You can think of `minLevelEdgeBot` as the uppermost level where cells on both sides are active. 

Conceptually, when there is topography under ice shelves and thus inactive top cells, do you think that these vertical mixing variables should be computed at the top level of a cell when it is **not** horizontally connected to its neighbor? If the answer is yes, then we should use `minLevelEdgeTop`, if no, `minLevelEdgeBot`.

Reply:
Thanks, @cbegeman! Yes, it makes sense to me now to use `minLevelEdgeBot` here as the vertical mixing should only be applied away from the boundaries.